pymc/logprob/censoring.py

```diff
@@ -238,3 +260,282 @@ def round_logprob(op, values, base_rv, **kwargs):
     from pymc.math import logdiffexp
 
     return logdiffexp(logcdf_upper, logcdf_lower)
+
+
+class FlatSwitches(Op):
```

Comment:
May be more intuitive?

```suggestion
class NestedEncodingSwitches(Op):
```